WooCommerce/Classes/POS/Controllers/PointOfSaleItemsController.swift

```diff
@@ -117,13 +117,13 @@ private extension PointOfSaleItemsController {
         allItems.append(contentsOf: uniqueNewItems)
         if allItems.isEmpty {
             itemsViewState = .init(containerState: .empty,
-                                   itemsStack: ItemsStackState(root: .loaded([]),
+                                   itemsStack: ItemsStackState(root: .loaded([], hasMoreItems: false),
                                                                itemStates: [:]))
         } else {
             let itemStates = itemsViewState.itemsStack.itemStates
                 .filter { allItems.contains($0.key) }
             itemsViewState = .init(containerState: .content,
-                                   itemsStack: ItemsStackState(root: .loaded(allItems),
+                                   itemsStack: ItemsStackState(root: .loaded(allItems, hasMoreItems: pagedItems.hasMorePages),
```

Comment:
Maybe can have a test case for this when `pagedItems.hasMorePages` is `true`, if not already? Asking because I was seeing all `hasMoreItems: false` in the `PointOfSaleItemsControllerTests` diffs.

Reply:
Yep, I'll add one